docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -79,11 +79,11 @@ This is essential, as it allows you to work on your own copy of freeCodeCamp on
 3. After the repository has been forked, you will be taken to your copy of the freeCodeCamp repository at `https://github.com/YOUR_USER_NAME/freeCodeCamp`
 
 <details>
-   <summary>
-      How to fork freeCodeCamp on GitHub (screenshot)
-   </summary>
-   <br>
-   <img src="https://raw.githubusercontent.com/freeCodeCamp/freeCodeCamp/master/docs/images/github/how-to-fork-freeCodeCamp.gif" alt="How to fork freeCodeCamp on GitHub">
+  <summary>
+     How to fork freeCodeCamp on GitHub (screenshot)
+  </summary>
```

Comment:
Some problem with `details` tag, not rendering properly
I tried but can't figure out what the problem is

Reply:
Ok